ingestion-lambda/src/handler.ts

```diff
@@ -1,17 +1,49 @@
 import type {
 	SESEvent,
+	SESEventRecord,
 	SQSBatchResponse,
 	SQSEvent,
 	SQSRecord,
 } from 'aws-lambda';
+import { getFromEnv, isRunningLocally } from '../../shared/config';
+import { findVerificationFailures } from '../../shared/findVerificationFailures';
 import { createLogger } from '../../shared/lambda-logging';
 import { initialiseDbConnection } from '../../shared/rds';
+import { FEEDS_BUCKET_NAME } from '../../shared/s3';
 import type { BatchItemFailure, OperationResult } from '../../shared/types';
 import { putItemToDb } from './db';
 import { getItemFromS3 } from './getItemFromS3';
-import { processContent } from './processContentObject';
+import { processFingerpostJsonContent } from './processContentObject';
+import { processEmailContent } from './processEmailContent';
 
-function processMessage(
+function processSESRecord(record: SESEventRecord): OperationResult<{
+	externalId: string;
+	objectKey: string;
+}> {
+	const { ses } = record;
+
+	const { hasFailures, failedChecks } = findVerificationFailures(ses.receipt);
```

Comment:
adding the verification failure checks here because the email filtering lambda hasn't been switched on properly yet, but when it has we can delegate to it if we want to